### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by ABfeMate**
- [DEMO LINK](https://ABfeMate.github.io/layout_snake/)
- [TEST REPORT LINK](https://ABfeMate.github.io/layout_snake/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  background-color: #600;
+}
+
+.block:nth-child(5) {
+  background-color: #300;
+}
+
+.block:nth-child(6) {
+  background-color: #000;
+}
```
Comment (Bohdan-Maksymiuk) [inline src/style.css:47]: you can use loop here instead of hardcoding

**Review by Bohdan-Maksymiuk — CHANGES_REQUESTED** [review]
looks good but fix demo links - they don't work
maybe you forgot run deploy?

File: src/style.css (hunk 1 above)
Comment (ABfeMate) [reply src/style.css:47]: I don't know how to use loops in css, so I'll leave it for later.

**Review by Bohdan-Maksymiuk — APPROVED** [review]
Good job 👍

**Pull request closed without merging**